modules/round/src/main/JsonView.scala

```diff
@@ -38,9 +38,10 @@ final class JsonView(
   private def kingMoves(game: Game, playerIndex: PlayerIndex) =
     (game.variant.frisianVariant) option game.history.kingMoves(playerIndex)
 
-  private def onlyDropsVariantForCurrentAction(pov: Pov): Boolean = {
-    pov.game.variant.onlyDropsVariant || (pov.game.variant.key == "amazons" && pov.game.situation.destinations.size == 0)
-  }
+  private def onlyDropsVariantForCurrentAction(pov: Pov): Boolean = pov.game.variant.onlyDropsVariant
+  // TODO: in analysis mode, this will be evaluated against the last move, but we don't want to set onlyDropsVariant
+  // in this case.
+  // pov.game.variant.onlyDropsVariant || (pov.game.variant.key == "amazons" && pov.game.situation.destinations.size == 0)
```

Comment:
I don't remember what this was for, I think I'll reevaluate it when I'm finalizing the amazons integration after the multimove stuff.